packages/module/src/Unavailable/Unavailable.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { EmptyState, EmptyStateBody, EmptyStateHeader, EmptyStateIcon, EmptyStateVariant } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { createUseStyles } from 'react-jss';
+import clsx from 'clsx';
+
+const useStyles = createUseStyles({
+  '.ins-c-empty-state__unavailable': {
+    color: 'var(--pf-global--danger-color--100)',
+    '& svg': { color: 'var(--pf-global--danger-color--100)' }
+  }
+});
+
+const Unavailable: React.FunctionComponent = () => {
+  const classes = useStyles();
+  return (
+    <EmptyState variant={EmptyStateVariant.lg} className={clsx(classes['.ins-c-empty-state__unavailable'], 'pf-m-redhat-font')}>
```

Comment:
'pf-m-redhat-font' probably should not be there

Reply:
done.